Bottom: 0f7d0508d7f04f2c277a3243cff503a97862859a
Top:    0198efdebb4d3ebe3a6ea3da0665c1fa093a63b6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 17:59:49 +0100

Change scroll-wheel bindings to match more GTK applications









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 593b737..2276bfc 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 }
 
 MainMenu =
